### PR TITLE
Ten-37 added endpoint for unrecognised userIds

### PR DESCRIPTION
**Pull request by Chrissystk**

### CODE REVIEW

File: app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/AccountCheckController.scala [hunk 1]
```diff
@@ -36,7 +36,9 @@ class AccountCheckController @Inject()(
 
   def accountCheck(redirectUrl: String): Action[AnyContent] = authAction.async {
     implicit request =>
-      logger.info(s"[AccountCheckController][accountCheck] User IP: ${request.headers.get("True-Client-IP")}")
+      logger.info(
+        s"[AccountCheckController][accountCheck] User IP: ${request.headers.get("True-Client-IP")}"
```
Comment (Ufuoma-Daniel) [inline app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/AccountCheckController.scala:40]: Can you delete this? I added this for some allow list testing when I was stuck, we likely don't need it anymore.

**Comment by Ufuoma-Daniel** [comment]
Won't this fail the pipeline? Coverage is below minimum [93.56% < 94.00%]

**Comment by Chrissystk** [comment]
> 

When I ran coverage it was 94.74?

File: app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/AccountCheckController.scala (hunk 1 above)
Comment (Chrissystk) [reply app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/AccountCheckController.scala]: deleted

**Comment by Ufuoma-Daniel** [comment]
> > 
> 
> When I ran coverage it was 94.74?

image
 The command I ran was - sbt clean coverage test it:test coverageReport

**Review by Ufuoma-Daniel — APPROVED** [review]

**Pull request merged**